packages/client/ui/react-ui/src/hooks/useCrossmint.tsx

```diff
@@ -56,7 +57,11 @@ export function CrossmintProvider({
         [setJwt, setRefreshToken, version]
     );
 
-    return <CrossmintContext.Provider value={value}>{children}</CrossmintContext.Provider>;
+    return (
+        <CrossmintContext.Provider value={value}>
```

Comment:
hmm i would go even deeper/more component specific than this

for ex, we wont be using twind for hosted or embedded checkout, so having it this high isnt optimal

Reply:
i'd be fine with it on CrossmintAuthProvider for now, or maybe just on the actual component

Reply:
definitely, i'll toss is lower 👍